Homework9/Homework9/Services/Abstractions/IFileService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Homework9.Services.Abstractions
+{
+	public interface IFileService
+	{
+        string GetDirectory();
+        void CreateDirectory();
+        void CheckIfDirectoryCouldBeCreated();
+        string SetLogFileName();
+        void WriteLogsToAFile();
+        string[] GetAllFilesFromDirectory();
+        string[] GetFilesWithoutOldestOne();
+        string[] SortFilesByDate(string[] files);
+        void DeleteOldestFile(string[] sortedFiles);
+        string ConvertLogsToJson();
+    }
```

Comment:
What about segregate this interface to two smaller like IFiseService and ILoggerService?